app/static/widgets/calculator.html

```diff
@@ -0,0 +1,249 @@
+<!--
+    Calculator widget.
+    This file should contain all required 
+    CSS, HTML, and JS for it.
+-->
+
+<style>
+    #calc-text {
+        background: var(--whoogle-dark-page-bg);
+        padding: 8px;
+        border-radius: 8px;
+        text-align: right;
+        font-family: monospace;
+        font-size: 16px;
```

Comment:
Should probably specify font color here as well, since otherwise the calculator text isn't visible for light themes. 
```suggestion
        font-size: 16px;
        color: var(--whoogle-dark-text);
```